draft-ietf-ppm-dap.md

```diff
@@ -1461,8 +1487,9 @@ processed further and MUST be removed from the candidate set.
 If the Helper's `PrepareStep` is of type `continued`, then the Leader proceeds
 as follows.
 
-Let `leader_outbound` denote the leader's prepare message and `helper_outbound`
-denote the helper's. The leader computes the next state transition as follows:
+Let `leader_outbound` denote the Leader's prepare share and `helper_outbound`
```

Comment:
I think you used the term "leader_prep_share" above; perhaps replace "leader_outbound" with "leader_prep_share" for consistency? (The new name is much better, IMO!)